src/About.css

```diff
@@ -0,0 +1,89 @@
+.main {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 100%;
+  min-height: 100vh;
+}
+
+.container {
+  display: flex;
+  width: 100%;
+  min-height: 100%;
+  justify-content: space-between;
+}
+
+.about {
+  width: 60%;
+  padding: 0 100px 0 30px;
+}
+
+.image {
+  background-image: url("./assets/img-about.jpg");
+  background-size: cover;
+  background-position: center center;
+  height: 100%;
+  width: 40%;
+}
+
+
+.about__title {
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: left;
+}
+
+.about__text {
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+}
+
+/*HELPER*/
+
+.mt-60 {
+  margin-top: 60px;
+}
+
+.mt-30 {
+  margin-top: 30px;
+}
+
+.pt-60 {
+  padding-top: 60px;
+}
+
+@media screen and (max-width: 390px) {
```

Comment:
if possible make it more adaptive for different devices
![Screenshot 2024-04-21 at 22 42 30](https://github.com/MastersAcademy/fe-react-2024/assets/115080771/0e98af59-e332-485f-86a9-59577d3f2fa0)